src/components/molecules/Loading/LoadingIcon.tsx

```diff
@@ -0,0 +1,22 @@
+export const LoadingIcon: React.FC = () => (
+  <svg
```

Comment:
[Q] Why isn't this and similar ones put as separate `.svg` files in the assets directory? Tailwind limitation?

Reply:
> Tailwind limitation?

yes, the SVG has the tailwind classes. And tailwind excludes classes from the final stylesheet if they are not found in the codebase.